File: BitManipulations/FindUniqueNumber.go
package main

 import (
	"fmt"
 )

 func main() {
 	a := [7]int{1,2,1,3,1,2,1}
 	var res int
	for i := 0; i < 7; i++ {
		res^=a[i]
	}
	fmt.Println(res)
 }

File: BitManipulations/FindUniqueNumber.py
def findUniqueNumber(a):
	"""Function to print unique number"""
	res = 0
	
	for i in range(len(a)):
		res = res  ^ a[i]
	
	print "Unique number is ", res, '.'

# decimal number
a=[1,2,1,3,1,2,1]

findUniqueNumber(a)
